lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,7 +16,7 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = {})
-          @root = (@options[:root] || serializer.json_key.to_s.pluralize).to_sym
+          @root = :data
```

Comment:
As this fixed by standard whould be better to move that into `initialize` method?